containers/kubernetes/samples/config-cm.yaml

```diff
@@ -73,10 +73,16 @@ data:
     # specify eviction strategy to be used when the storage size is achieved
     ZWE_CACHING_EVICTION_STRATEGY=reject
     # specify persistent method of caching service
-    # possible options: <empty> (will fall back to inMemory), VSAM, redis
-    ZWE_CACHING_SERVICE_PERSISTENT=
+    # possible options: <empty> (will fall back to inMemory), VSAM, redis, infinispan
+    ZWE_CACHING_SERVICE_PERSISTENT=infinispan
     # specify the data set name of the caching service VSAM data set
     ZWE_CACHING_SERVICE_VSAM_DATASET=IBMUSER.ZWE.CACHE1
+    # specify the jgroup bind port for infinispan
+    ZWE_CACHING_SERVICE_INFINISPAN_JGROUPS_BIND_PORT=7098
+    # specify the path where the Soft-Index store will keep its data files for the Infinispan Soft-Index Cache Store
+    ZWE_CACHING_SERVICE_INFINISPAN_PERSISTENCE_DATALOCATION=data
+    # Infinispan/Jgroup TCPPING property that specifies the list of cluster nodes (members)
+    ZWE_CACHING_SERVICE_INFINISPAN_INITIALHOSTS="${ZOWE_EXPLORER_HOST}[${ZWE_CACHING_SERVICE_INFINISPAN_JGROUPS_BIND_PORT}]"
```

Comment:
How the infinispan instances will be started? are they separated processes? I wonder how this will work in k8s deployment.

In k8s deployment, gateway/discovery/api-catalog/caching-service will be started in separate containers. Will the infinispan process be part of caching-service? Which means, if we have multiple caching-service instances, it will have one infinispan instance for each of them?

Reply:
Hey Jack, thanks for taking a look. The infinispan is part of the caching service, to it will be started together with it. In case of multiple caching service, though, the configuration is a bit different. In my case, the sanity tests are not executed at all, it rather fails at the very beginning seems like. Not sure why.